src/portfft/committed_descriptor_impl.hpp

```diff
@@ -286,12 +297,10 @@ class committed_descriptor_impl {
         if (detail::can_cast_safely<IdxGlobal, Idx>(factor_sg) && detail::can_cast_safely<IdxGlobal, Idx>(factor_wi)) {
           std::size_t input_scalars =
               num_scalars_in_local_mem(detail::level::SUBGROUP, static_cast<std::size_t>(factor_size), SubgroupSize,
-                                       {static_cast<Idx>(factor_sg), static_cast<Idx>(factor_wi)}, temp_num_sgs_in_wg,
+                                       {static_cast<Idx>(factor_wi), static_cast<Idx>(factor_sg)}, temp_num_sgs_in_wg,
```

Comment:
This was a bug, subgroup's num_Scalar in local mem takes in the factor array in order factor_wi, factor_sg, but here it was being passed as factor_sg, factor_wi. Luckily we never ran into this issue because local memory was always being computed for the BATCH_INTERLEAVED format which multiplies the two values anyways, hence it was fine until now.